### ISSUE TITLE
My model uses bilinear in UPSAMPLE，is it possible that I can modify your code to fit mine？

### ISSUE THREAD

**Issue by yw2278** [xxradon/PytorchToCaffe#35]
Is there any hint to do that？

**Comment by xxradon** [comment]
Yes，you should modify my code to fit your own caffe model.as we known,caffe have so many modified editions,I just implement a few operations(layers).

**Comment by yw2278** [comment]
Alright，Thank You！

**Comment by yw2278** [comment]
Sorry, just a quick question do I need to change code in Caffe or I canjust modify pytorch_to_caffe and layer_param.py?Thank you!

**Comment by xxradon** [comment]
You should know that this [caffe.proto](https://github.com/xxradon/PytorchToCaffe/blob/d486db2e1a759f0a4315e943606c937113c3a96d/Caffe/caffe.proto#L1)  should be the same with your our caffe,and if you want to add new layer,yes you should modify pytorch_to_caffe.py and layer_param.py.

**Comment by yw2278** [comment]
Sorry to bother you again, but could you further explain the meaning " caffe.proto should be the same with your our caffe"? Is caffe_pb2.py generated from caffe.proto?

**Comment by xxradon** [comment]
yes,it is.Because caffe have so much editions,and even the same layer have different parameters,so you should make sure the caffe proto is your own caffe.proto.

**Comment by imistyrain** [comment]
You can use Deconvolution to implement bilinear upsample with suitable weights.

**Comment by wanglaotou** [comment]
> You can use Deconvolution to implement bilinear upsample with suitable weights.

hi, i use deconvolution replace the bininer unsample, but i met other problem when i want to transform the pytorch to caffe model. could you please tell me more details.
my modification is:
![图片](https://user-images.githubusercontent.com/12837585/144703871-6196db40-fa5c-470c-8bca-0cd43f7cc181.png)
